challenges/desafio1.js

```diff
@@ -0,0 +1,10 @@
+db.movies.aggregate([
+  {
+    $match: {
+      "imdb.rating": { $gte: 7 },
+      $and: [{ languages: "English" }, { languages: "Spanish" }],
+      $nor: [{ genres: "Crime" }, { genres: "Horror" }],
+      $or: [{ rated: "PG" }, { rated: "G" }],
```

Comment:
Ótima estrutura!! Eu me confundo muuito pra usar o and, o nor e o or, então usei o in, nin e all. Só vou te mostrar como eu fiz: 
```suggestion
      languages: { $all: ["English", "Spanish"] },
      genres: { $nin: ["Crime", "Horror"] },
     rated: { $in: ["PG", "G"] },
```